statsmodels/tsa/statespace/varmax.py

```diff
@@ -815,7 +816,7 @@ def predict(self, start=None, end=None, exog=None, dynamic=False,
                         kwargs[name] = mat[:, :, -_out_of_sample:]
         elif self.model.k_exog == 0 and exog is not None:
             warn('Exogenous array provided to predict, but additional data not'
-                 ' required. `exog` argument ignored.')
+                 ' required. `exog` argument ignored.', InvalidInputWarning)
```

Comment:
What's the difference here between InvalidInputWarning and InputIgnoredWarning, given that you also added the latter